src/fheroes2/battle/battle_only.cpp

```diff
@@ -462,6 +462,11 @@ void Battle::Only::RedrawBaseInfo( const fheroes2::Point & top ) const
     }
 
     fheroes2::RedrawPrimarySkillInfo( top, armyInfo[0].ui.primarySkill.get(), armyInfo[1].ui.primarySkill.get() );
+
+    // hide the swap army/artifact arrows
+    const fheroes2::Sprite & moveButtonBackground = fheroes2::AGG::GetICN( ICN::STONEBAK, 0 );
+    fheroes2::Blit( moveButtonBackground, 292, 270, display, top.x + 292, top.y + 270, 48, 44 );
+    fheroes2::Blit( moveButtonBackground, 292, 363, display, top.x + 292, top.y + 363, 48, 44 );
```

Comment:
Since the original image (STONEBAK) has no shadows or transparency, we can use `Copy()` instead of `Blit()` to speed up rendering a bit.
```suggestion
    fheroes2::Copy( moveButtonBackground, 292, 270, display, top.x + 292, top.y + 270, 48, 44 );
    fheroes2::Copy( moveButtonBackground, 292, 363, display, top.x + 292, top.y + 363, 48, 44 );
```

Reply:
No, `Blit()` behaves as `Copy()` only when handling single-layer images. By default images are double-layer images. Therefore, the complexity of the operation increases leading to slower execution.